### PR TITLE
Front end development initial

### PR DESCRIPTION
**Pull request by Gabriel-Ivarsson**
Update:
- Front-end server made to run in the front-end auto-scaling group

### CODE REVIEW

**Comment by NoahHakansson** [comment]
Will the static content still be served by a GO server, or was this changed to be handled elsewhere?

File: frontend/go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module front-end-cloud-calc
+
+go 1.19
```
Comment (NoahHakansson) [inline frontend/go.mod:3]: Downgrade to 1.18 to be the same as backend  and for Devs who don't have 1.19 yet

**Review by NoahHakansson — APPROVED** [review]
Didn't see any logic in the frontend to send request to the API. Maybe I missed it or it's just not implemented yet?

File: frontend/go.mod (hunk 1 above)
Comment (Gabriel-Ivarsson) [reply frontend/go.mod:3]: Too slow

**Pull request merged**